.github/actions/wasm-ci/action.yml

```diff
@@ -16,7 +16,7 @@ runs:
         -DF3D_MODULE_UI=OFF
         -DF3D_PLUGIN_BUILD_ASSIMP=ON
         -DF3D_PLUGIN_BUILD_DRACO=ON
-        -DF3D_PLUGIN_BUILD_EXODUS=OFF
+        -DF3D_PLUGIN_BUILD_HDF=OFF
```

Comment:
turns out wasm supports hdf, so stay tuned, you should be able to turn that on once I rebuild the image (see https://github.com/f3d-app/f3d-docker-images/pull/40)

Reply:
(and add the related extension to supported F3DWeb stuff)

Reply:
nice!

Reply:
alright, you should now be able to enable HDF plugin on wasm after a rebase on master